dev-haskell/pointfree/pointfree-1.1.1.10.ebuild

```diff
@@ -23,3 +23,10 @@ DEPEND="${RDEPEND}
 	test? ( >=dev-haskell/hunit-1.6 <dev-haskell/hunit-1.7
 		>=dev-haskell/quickcheck-2.11 <dev-haskell/quickcheck-2.15 )
 "
+
+src_prepare() {
+	default
+
+	cabal_chdeps \
+		'base >= 4.5 && < 4.17' 'base >= 4.5 && < 5'
+}
```

Comment:
We really shouldn't be using `cabal_chdeps` like this anymore. `src_prepare` handles some extra stuff now, such as running `dos2unix` on cabal files and bringing in Hackage revision bumps. This stuff will get skipped if `src_prepare` is overridden and `default` is used.

I marked `cabal_chdeps` as deprecated in the eclass a little while back. Use `CABAL_CHDEPS` instead.

Reply:
yes I forgot